libvast/src/system/index.cpp

```diff
@@ -356,9 +356,11 @@ caf::error index_state::load_from_disk() {
                 std::exchange(synopses, {}))
       .then(
         [this](atom::ok) {
-          VAST_VERBOSE("{} successfully loaded meta index from disk and will "
-                       "start processing queries",
-                       *self);
+          VAST_DEBUG("{} successfully loaded meta index from disk and will "
+                     "start processing queries",
+                     *self);
+          VAST_INFO("{} finished initalizing and is ready to accept queries",
```

Comment:
If we have an INFO message, I don't think we still need a DEBUG message in the same place.

What's the general idea here, is it to be able to see the message from `systemctl status`?

Reply:
I didn't delete the old message because it contains more information but of course the new one implies the same for those in the know. The new message is worded differently to avoid confusing regular users.

I have now problem with removing the debug message.